packaging/build-docker-centos-rpm.sh

```diff
@@ -0,0 +1,62 @@
+RELEASE_VERSION=$1
+if [ -z ${RELEASE_VERSION} ]
+then
+  echo "You need so specify the release version you wish to build: e.g './build-docker-centos-rpm.sh 1.12.0'"
+  echo "https://github.com/jembi/openhim-console/releases"
+  exit
+fi
+
+# Set docker container name to build RPM package
+containerName=openhim-console-centos-rpm
+
+# Define the CentOS version to build in the docker container
+docker pull centos:7
+
+docker run -t -d --rm --name $containerName centos:7 /bin/bash
+
+echo "Update packages: "
+docker exec -it $containerName sh -c "yum -y update"
+
+echo "Install needed packages: "
+docker exec -it $containerName sh -c "yum install -y git rpm-build redhat-rpm-config gcc-c++ make"
+
+echo "Install needed packages: "
+docker exec -it $containerName sh -c "curl --silent --location https://rpm.nodesource.com/setup_10.x | bash -"
+
+echo "Install needed packages: "
+docker exec -it $containerName sh -c "yum install -y nodejs"
+
+echo "Fetch release version from Github"
+docker exec -it $containerName sh -c "mkdir /openhim-console && curl -sL 'https://github.com/jembi/openhim-console/archive/v$RELEASE_VERSION.tar.gz' | tar --strip-components=1 -zxv -C /openhim-console"
```

Comment:
@MattyJ007 there is a way to achieve this but will require some refactoring getting it to work properly, I dont think its worth adding in now as the person building the package should know the existing release they are trying to build. This could be added in future if its really becomes an issue trying to build packages that doesnt exist